Prashant Kawatkar/Assignments/cypress/integration/assignments/GreenCart.js

```diff
@@ -0,0 +1,37 @@
+describe('Perform testing for Greencart application', function() 
+{
+ 
+it('Test case for Greencart application',function() {
+
+cy.visit("https://rahulshettyacademy.com/seleniumPractise/#/")
```

Comment:
Make is configurable in Cypress.config.json

Reply:
Done